AK/Concepts.h

```diff
@@ -36,6 +37,9 @@ concept SameAs = IsSame<T, U>;
 template<typename T>
 concept AnyString = Detail::IsAnyString<Detail::Decay<T>>;
 
+template<typename T, typename U>
+concept HashCompatible = IsHashCompatible<Detail::Decay<RemoveCVReference<T>>, Detail::Decay<RemoveCVReference<U>>>;
```

Comment:
`RemoveCVReference` is redundant here, no?

Reply:
Ah your decay impl doesn't remove references. The `RemoveCV` could be changed to `RemoveCVReference`. The STL's `std::decay` removes references as well.